src/infra/sequelize/queries/frise/getGarantiesFinancièresDTO.integration.ts

```diff
@@ -0,0 +1,178 @@
+import { getGarantiesFinancièresDTO } from './getGarantiesFinancièresDTO'
+import { resetDatabase } from '@infra/sequelize/helpers'
+import { UniqueEntityID } from '@core/domain'
+import { User } from '@entities'
+import { GarantiesFinancièresStatut } from '@infra/sequelize/projectionsNext/garantiesFinancières'
+
+describe(`Requête getGarantiesFinancièresDTO`, () => {
+  const utilisateurAutorisé = { role: 'admin' } as User
+  const dateLimiteEnvoi = new Date('2023-01-01')
+  const dateEchéance = new Date()
+  const envoyéesPar = new UniqueEntityID().toString()
+  const dateConstitution = new Date()
+  const fichierId = new UniqueEntityID().toString()
+
+  beforeEach(async () => await resetDatabase())
+
+  describe(`Retourner les données de GF en retard`, () => {
+    it(`Etant donné un projet soumis à garanties financières,
+  et dont les GF sont en attente avec une date limite d'envoi dépassée,
+  alors la requête devrait retourner un GarantiesFinancièresDTO 
+  avec un statut 'en en retard'`, async () => {
+      const dateLimiteDépassée = new Date('2021-01-01')
+      const garantiesFinancières = {
+        statut: 'en attente' as GarantiesFinancièresStatut,
+        soumisesALaCandidature: false,
+        dateLimiteEnvoi: dateLimiteDépassée,
+        envoyéesPar: new UniqueEntityID().toString(),
+        dateConstitution: null,
+        dateEchéance: null,
+        validéesPar: null,
+      }
```

Comment:
Alternative plus safe et qui se passe d'un import:
```suggestion
      const garantiesFinancières = {
        statut: 'en attente',
        soumisesALaCandidature: false,
        dateLimiteEnvoi: dateLimiteDépassée,
        envoyéesPar: new UniqueEntityID().toString(),
        dateConstitution: null,
        dateEchéance: null,
        validéesPar: null,
      }  as const
```